packages/firebase_crashlytics/README.md

```diff
@@ -91,20 +122,21 @@ void main() {
 }
 ```
 
-Overriding `FlutterError.onError` with `Crashlytics.instance.recordFlutterError`  will automatically catch all 
-errors that are thrown from within the Flutter framework.  
-If you want to catch errors that occur in `runZoned`, 
-you can supply `Crashlytics.instance.recordError` to the `onError` parameter:
+Overriding `FlutterError.onError` with `Crashlytics.instance.recordFlutterError`  will automatically catch all errors that are thrown from within the Flutter framework.
+
+If you want to catch errors that occur in `runZoned`, you can supply `Crashlytics.instance.recordError` to the `onError` parameter:
+
 ```dart
-runZoned<Future<void>>(() async {
+runZonedGuarded<Future<void>>(() async {
```

Comment:
Double check the code snippet, onError is not a named argument for `runZonedGuarded`. The text above that describes the code will need to be updated as well

Reply:
Thanks! I missed that